src/features/Chat/Chat.tsx

```diff
@@ -0,0 +1,44 @@
+import styled, { css } from 'styled-components';
+import { useState } from 'react';
+
+import ChatMenu from './ChatMenu';
+import ChatWindow from './ChatWindow';
+import PageWithTransition from '@/components/PageWithTransition';
+import Spinner from '@/components/Spinner';
+
+import { useMobileMode } from '@/hooks/useMobileMode';
+
+const ChatPage = () => {
+  const isMobile = useMobileMode();
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
I would not use useState, when we dont set state, even though it is a placeholder. Instead I would just use a const

Reply:
Fixed!